### PR TITLE
Remove unused variables from gpstart

### PR DESCRIPTION
**Pull request by danielgustafsson**
These variables are since commit 4eaeb7bc2ce no longer used by any caller, remove.

ping @ashwinstar

### CODE REVIEW

File: gpMgmt/bin/gpstart [hunk 1]
```diff
@@ -706,8 +706,6 @@ class GpStart:
                 host = self.gparray.standbyMaster.hostname
                 datadir = self.gparray.standbyMaster.datadir
                 port = self.gparray.standbyMaster.port
-                dbid = self.gparray.standbyMaster.dbid
-                ncontents = self.gparray.getNumSegmentContents()
```
Comment (ashwinstar) [inline gpMgmt/bin/gpstart]: Ideally, I wished to do next round of cleanup as think `gparray.getNumSegmentContents()` shouldn't be used at all now in utilities. So, please feel free to take that action as part of this PR, or I can make follow PR to it later.

**Comment by berlin-ab** [comment]
Nice find. +1

File: gpMgmt/bin/gpstart (hunk 1 above)
Comment (danielgustafsson) [reply gpMgmt/bin/gpstart]: Available time, combined with existing Python skills, is a bit a low at the moment so I've opted for just pushing this leaving further cleanup for another PR.

**Comment by danielgustafsson** [comment]
Pushed to master, thanks for review!

**Pull request closed without merging**